resources/views/themes/default1/front/auth/login-register.blade.php

```diff
@@ -1481,9 +1482,12 @@ function getCountryAttr(val) {
 
         function getState(val) {
             $.ajax({
+                headers: {
+                   'X-CSRF-TOKEN': $('meta[name="csrf-token"]').attr('content')
+                },
                 type: "GET",
                 url: "{{url('get-loginstate')}}/" + val,
-                data: {'country_id':val,'_token':"{{csrf_token()}}"},//'country_id=' + val,
+                data: {'country_id':val},//'country_id=' + val,
```

Comment:
@sowmiyasubramaniyan you have to csrf token to the form. Not just remove it from the URL. Now this is vulnerable to cross-site attack